Dockerfile

```diff
@@ -40,6 +40,3 @@ RUN bundle install --jobs 20 --retry 5
 
 # Copy the main application.
 COPY . ./
-
-# Default command
-ENTRYPOINT ['/app/robles/bin/robles']
```

Comment:
I had to get rid of this so that local development would work. It's not actually working as expected in CI anyway.